lib/sdf-server/src/service/v2/admin.rs

```diff
@@ -109,6 +126,8 @@ pub enum AdminAPIError {
     WorkspaceSnapshotAddressNotFound(ChangeSetId),
     #[error("workspace snapshot {0} for change set {1} could not be found in durable storage")]
     WorkspaceSnapshotNotFound(WorkspaceSnapshotAddress, ChangeSetId),
+    #[error("ws event error: {0}")]
+    WsEvent(#[from] WsEventError),
```

Comment:
nit: where we can, we should qualify the error namespaces inline here--no need to specify them multiple times, and it makes them more copy/pasteable.